ui/src/publication/split/split-details-dialog.scss

```diff
@@ -0,0 +1,13 @@
+@use 'vayla-design-lib' as vayla-design;
+
+.split-details-dialog__table {
+    height: 250px;
+    overflow: auto;
+    margin-bottom: 20px;
+
+    &-header {
```

Comment:
Olikohan meillä näille tyylinimitysyhdistelyille jotain suuntaviivaa?

Omana kokemuksena varsinaisen tyylin määritelmän löytäminen/siihen hyppääminen on näiden yhdistelyjen seurauksena usein aika aikaavievää varsinaisesta komponenttikoodista, kun suoralla komponentista löytyvällä nimityksellä ei voi hakea varsinaista kohtaa tyylitiedostosta. Tässä toki probleema on pienempi, koska scss-tiedosto sisältää vain tämän tyylin. 

Pitäisi varmaan selvitellä saisiko Idean hyppäämään suoraan tänne ctrl+klikillä komponentin tyylin nimityksestä, eikä typescriptin tyyppitiedostoon, johon se nyt hyppäilee.

Reply:
_Yhdistelyille_ ei taida olla mitään omia sääntöjä/suuntaviivoja, vaan jäänyt pääasiassa kehittäjän omalle päätösvallalle. BEM:hän meillä on käytössä nimeämiskäytäntönä tyyleille ja monesti ollaan tehty näitä yhdistelyjä sen "osasten" rajoilla, mutta tämä yhdistäminen ei tosiaan niitä rajoja kunnioita. Tuollainen parempi IDE-tuki olisi kyllä mahtava jos sellainen jotenkin saataisiin, itsekin olen noita miljoonaan kertaan kironnut. Tuo TypeScriptin tyyppitiedostojuttukin on sikäli ikävä, että muistaakseni sen tuottaman paketin kanssa oli jotain ongelmia eikä pakettiin ole tullut enää päivityksiä (olisiko ollut jotain transitiivisia tietoturvajuttuja, jotka eivät ole meillä käyttötavasta johtuen mikään oikea ongelma koska kyseessä on `devDependency`.) Sen korvaamista voisi siis todellakin harkita jollain paremmalla ratkaisulla joskus.